client/components/shared/CallCentreAccordion.stories.tsx

```diff
@@ -1,9 +1,32 @@
 import type { Meta, StoryObj } from '@storybook/react';
+import { userEvent, within } from '@storybook/testing-library';
 import { CallCentreAccordion } from './CallCentreAccordion';
 
+const sleep = (ms: number) => new Promise((r) => setTimeout(r, ms));
+
 export default {
 	title: 'Components/CallCentreAccordion',
 	component: CallCentreAccordion,
 } as Meta<typeof CallCentreAccordion>;
 
-export const Default: StoryObj = {};
+export const Closed: StoryObj = {};
+
+export const Open: StoryObj = {
+	args: { showEmailAddress: true },
+	play: async ({ canvasElement }: { canvasElement: HTMLElement }) => {
+		// There is a useEffect within AccordionRow that sets 'isBrowser' that isn't being run in storybook
```

Comment:
I had to do some investigation to figure out why this didn't work, and found the `sleep(1)` solution on this [issue](https://github.com/storybookjs/storybook/issues/18258). A bit of a hacky workaround so maybe won't use lots of `play` functions 😞

Reply:
It is a wee bit but I like it! We could add an additional code comment explaining the sleep(1) is required due a bug....